samri/optimization/registration.py

```diff
@@ -96,8 +97,8 @@ def structural_per_participant_test(participant,
 			res = struct_registration.run()
 
 def structural_rigid_affine(template="/Users/marksm/GitHub/mriPipeline/ants_test/template4.nii.gz",
```

Comment:
Looking good - so this function here now uses FLIRT for the reorient, and ANTs for everything else? Perhaps we should edit the name accordingly? and add a short docstring stating as much?